addons/web_editor/static/src/components/upload_progress_toast/upload_service.js

```diff
@@ -106,7 +106,20 @@ export const uploadService = {
                 // limited by bandwidth.
                 for (const sortedFile of sortedFiles) {
                     const file = progressToast.files[sortedFile.progressToastId];
-                    const dataURL = await getDataURLFromFile(sortedFile);
+                    let dataURL;
+                    try {
+                        dataURL = await getDataURLFromFile(sortedFile);
+                    } catch {
+                        deleteFile(file.id);
+                        env.services.notification.add(
+                            _.str.sprintf(
```

Comment:
Could you use the odoo `sprintf` from `odoo/addons/web/static/src/core/utils/strings.js` instead of `_` library?

Reply:
Done changes :+1: